python/kserve/kserve/inference_client.py

```diff
@@ -107,6 +108,7 @@ def __init__(
         creds: grpc.ChannelCredentials = None,
         channel_args: List[Tuple[str, Any]] = None,
         timeout: Optional[float] = 60,
+        retries: Optional[int] = 3,
```

Comment:
I think it is better to default to 0 for calling predictor as the client side might already have configured the retries